src/test/java/com/sap/sgs/phosphor/fosstars/data/github/HasSecurityPolicyTest.java

```diff
@@ -1,73 +1,91 @@
 package com.sap.sgs.phosphor.fosstars.data.github;
 
-import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.SCANS_FOR_VULNERABLE_DEPENDENCIES;
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.HAS_SECURITY_POLICY;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
-import static org.junit.Assume.assumeNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
-import com.sap.sgs.phosphor.fosstars.data.Terminal;
 import com.sap.sgs.phosphor.fosstars.model.Value;
 import com.sap.sgs.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.sgs.phosphor.fosstars.tool.github.GitHubDataFetcher;
 import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProject;
+import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProjectValueCache;
 import java.io.IOException;
 import java.util.Optional;
+import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
+import org.kohsuke.github.GHContent;
+import org.kohsuke.github.GHRepository;
 import org.kohsuke.github.GitHub;
 
-/**
- * TODO: This test fails when GitHub API rate limit exceeded or some networking issues occur.
- *       The test needs to be re-written to use mocks.
- *       Meanwhile, the test is ignored.
- */
-@Ignore
 public class HasSecurityPolicyTest {
 
-  private static GitHub github;
-
   @BeforeClass
-  public static void initGithubConnection() {
-    try {
-      github = GitHub.connectAnonymously();
-      github.checkApiUrlValidity();
-    } catch (Exception e) {
-      System.out.println("Couldn't connect to GitHub!");
-      e.printStackTrace();
-      github = null;
-    }
+  public static void init() {
+    GitHubDataFetcher.instance().repositoryCache.clear();
+    GitHubDataFetcher.instance().commitsCache.clear();
   }
 
   @Test
-  public void testApacheNifi() throws IOException {
-    assumeNotNull(github);
+  public void testHasPolicy() throws IOException {
+    GHContent content = mock(GHContent.class);
+    when(content.isFile()).thenReturn(true);
+    when(content.getSize()).thenReturn(1000L);
+
+    GHRepository repository = mock(GHRepository.class);
+    when(repository.getFileContent("SECURITY.md")).thenReturn(content);
+
+    GitHub github = mock(GitHub.class);
+    when(github.getRepository(any())).thenReturn(repository);
+
+    GitHubDataFetcher fetcher = GitHubDataFetcher.instance();
+    fetcher.repositoryCache.clear();
```

Comment:
The @Before method would take care of this, before every test method execution? Hence, this may not be necessary.

Reply:
That's right. I'll remove it.

Reply:
Done.